specs/laravel/laravel_cache_table.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: Generate cache table

```

Comment:
nit: could we make this name specific to the type of workflow so that when someone searches for a workflow within the app it's clear when this should be run?

Reply:
Alright

Reply:
Yh, I think that would be great. Would update now

Reply:
Update done